CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- [#19] Redesign dogu-starting-page
```

Comment:
Like in cloudogu/nginx#103:  When we activly translate all text we should also add this to the change log